alphadia/workflow/optimization.py

```diff
@@ -101,17 +101,23 @@ def __init__(
             self.parameter_name
         ]["automatic_update_percentile_range"]
 
-        self.favour_narrower_parameter = workflow.config["optimization"][
-            self.parameter_name
-        ]["favour_narrower_parameter"]
+        self.try_narrower_values = workflow.config["optimization"][self.parameter_name][
+            "try_narrower_values"
+        ]
 
-        if self.favour_narrower_parameter:
+        if self.try_narrower_values:
             self.maximal_decrease = workflow.config["optimization"][
                 self.parameter_name
             ]["maximal_decrease"]
-            self.minimum_proportion_of_maximum = workflow.config["optimization"][
+
+        self.favour_narrower_optimum = workflow.config["optimization"][
+            self.parameter_name
+        ]["favour_narrower_optimum"]
+
+        if self.favour_narrower_optimum:
+            self.maximum_decrease_from_maximum = workflow.config["optimization"][
```

Comment:
```
self.maximum_decrease_from_maximum = workflow.config["optimization"][
                self.parameter_name
            ]["maximum_decrease_from_maximum"] if self.favour_narrower_optimum else None
```
(otherwise, the class would have different attribute fields depending on config .. (check also above)